discuss/customizations/call_notes_main_functionality.rst

```diff
@@ -0,0 +1,88 @@
+================
+Note Activitiy
+================
+
+New activity type called *Note* added next to currently 
+available generic *Other* type and *Meeting* (that allows 
+to create meeting from activity).
+
+If you select *Note* category on your activity type (most likely 
+on Call activity type), it will behave as a goal activity, where you 
+will need to select goal and contact. 
+
+Configuration
+=============
+
+To start using this new activity category, module :code:`mail_activity_note`
+must be installed and *Note* activity type added.
+
+To add this new activity type, go to :menuselection:`CRM --> Configuration --> Activity Types`
+and create  new activity type with category *Note*:
+
+.. image:: media/note_activity_category.png
+    :align: center
+
+Contact can be made mandatory on activity type, by checking *Contact required* option. 
+If not checked, it will be optional.
+
+*Hide popover* is alvays mandatory. This part must depend on 
+module :code:`mail_activity_popover_toggle` that allows to hide 
+little activity popup form. When *Note* category is chosen, 
+by default, it will select that option. If User force unselects it 
+when *Note* category is chosen, constraint will prevent from saving such changes.
+
+On activity type, one can specify partner field, so it would be possible to 
+know how to filter contacts on activity - *Path To Commercial Partner* field is used 
+for this purpose - add *partner_id.commercial_partner_id* in this field to 
+filter partner contacts on activity.
+
+Also *Auto Schedule Next Activity* option may be checked and *Default Next Activity* 
```

Comment:
This should be removed, because it's not related with this feature.

Reply:
Done